frontend/src/periscope/components/LineClampedText/LineClampedText.tsx

```diff
@@ -42,11 +44,25 @@ function LineClampedText({
 		</div>
 	);
 
-	return isOverflowing ? <Tooltip title={text}>{content}</Tooltip> : content;
+	return isOverflowing ? (
+		<Tooltip
+			title={text}
+			placement="right"
+			autoAdjustOverflow
```

Comment:
make sure this is not affecting the already existing instances of this

Reply:
+1

Reply:
thanks for catching it, actually I forgot to remove these, after passing` ...tooltipProps`